src/foam/test/CSVSinkDemo.js

```diff
@@ -81,8 +79,8 @@ foam.CLASS({
             str: 'Random data ' + i,
             bool: Math.random() < 0.5,
             arr: [Math.random()],
-            fop: this.Phone.create({ number: '111-111-1111' })
-          }));
+            fop: '111-111-1111' })
```

Comment:
```suggestion
            fop: '111-111-1111'
          }));
```

and remove the line below.

Reply:
Ok thanks